.github/workflows/chaos.yaml

```diff
@@ -54,7 +54,7 @@ jobs:
       uses: chaos-mesh/chaos-mesh-actions@master
       env:
         CFG_BASE64: YXBpVmVyc2lvbjogY2hhb3MtbWVzaC5vcmcvdjFhbHBoYTEKa2luZDogUG9kQ2hhb3MKbWV0YWRhdGE6CiAgbmFtZTogcG9kLWZhaWx1cmUKICBuYW1lc3BhY2U6IHBpbmdjYXAKc3BlYzoKICBhY3Rpb246IHBvZC1mYWlsdXJlIAogIG1vZGU6IG9uZSAKICBkdXJhdGlvbjogIjYwcyIKICBzZWxlY3RvcjoKICAgIGxhYmVsU2VsZWN0b3JzOgogICAgICAiYXBwLmt1YmVybmV0ZXMuaW8vY29tcG9uZW50IjogImNvbnRyb2xsZXItbWFuYWdlciIKICBzY2hlZHVsZXI6CiAgICBjcm9uOiAiQGV2ZXJ5IDVtIgo=
-        CHAOS_MESH_VERSION: v1.0.0
+        CHAOS_MESH_VERSION: v2.0.0
```

Comment:
Seems there're some issue in CI related to this change:
https://github.com/pingcap/tidb-operator/actions/runs/5586488291/jobs/10221058201#step:7:70
```console
 Error: INSTALLATION FAILED: unable to build kubernetes objects from release manifest: [resource mapping not found for name: "chaos-mesh-mutation" namespace: "" from "": no matches for kind "MutatingWebhookConfiguration" in version "admissionregistration.k8s.io/v1beta1"
```

Reply:
The version bump in chaos-mesh is in https://github.com/chaos-mesh/chaos-mesh/pull/2292, and `v2.1.0` should include that change. I think the easy way get that change in is to update chaos-mesh version to `v2.1.0` here.

Reply:
```suggestion
        CHAOS_MESH_VERSION: v2.1.0
```